packages/core/entity/FileDetail/RenderZarrThumbnailURL.ts

```diff
@@ -0,0 +1,101 @@
+import * as zarr from "@zarrita/core";
+import { FetchStore } from "@zarrita/storage";
+import { get as zarrGet } from "@zarrita/indexing";
+
+interface AxisData {
+    name: string;
+    type: "time" | "channel" | "space";
+    unit?: string;
+}
+
+// Define the type for the transformed array items
+interface TransformedAxes {
+    name: string;
+    value: number | null;
+}
+
+// Function to transform the array
+function transformAxes(originalArray: AxisData[]): TransformedAxes[] {
+    return originalArray.map((item) => ({
+        name: item.name,
+        value: item.type !== "space" ? 0 : null,
+    }));
+}
+
+export async function renderZarrThumbnailURL(zarrUrl: string): Promise<string> {
+    console.log("RENDERING THUMB");
```

Comment:
console log left in

Reply:
Ah yes we do not need to know "RENDERING THUMB"

Reply:
resolved in [552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)[552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)[552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)